### ISSUE TITLE
BERT_NER.py#L450 same code in if-else in both branches

### ISSUE THREAD

**Issue by natasasdj** [kyzhouhzau/BERT-NER#91]
BERT_NER.py#L450

`        if FLAGS.crf:
            (total_loss, logits,predicts) = create_model(bert_config, is_training, input_ids,
                                                            mask, segment_ids, label_ids,num_labels, 
                                                            `use_one_hot_embeddings)`

`        else:
            (total_loss, logits, predicts) = create_model(bert_config, is_training, input_ids,
                                                            mask, segment_ids, label_ids,num_labels, 
                                                            use_one_hot_embeddings)`

The same code is for both conditions.
